lib/mercenary/command.rb

```diff
@@ -227,7 +228,7 @@ def execute(argv = [], config = {})
     #
     # Returns true if this command is the parent of a command of name
     # 'sub_command' and false otherwise
-    def has_command?(sub_command)
+    def command?(sub_command)
```

Comment:
IMO Rubocop shouldn't be auto-correcting public methods.. Agreed, the library hasn't released a `v1.0` yet, but still I feel this shouldn't be a part of this PR..

Reply:
I did this change, as it don't break the tests, I assumed it was OK.